test/e2e/e2e_test.go

```diff
@@ -81,11 +81,11 @@ var _ = Describe("Workload cluster creation", func() {
 			}, result)
 			cluster = result.Cluster
 			targetCluster = bootstrapClusterProxy.GetWorkloadCluster(ctx, namespace, clusterName)
-			remediation()
+			testRemediation()
```

Comment:
revert please

Reply:
this is how it was in the PR, but I can replace it

Reply:
Yes, please